src/App/App.js

```diff
@@ -1,31 +1,22 @@
 import React from 'react';
 import { BrowserRouter, Route } from 'react-router-dom';
 
-
 import Header from '../Components/Header/header';
 import NavBar from '../Components/Navbar/nav';
-import Profile from '../Components/Profile/Profile';
-import Dialogs from '../Components/Dialogs/dialogs';
-import News from '../Components/News/news';
-import Music from '../Components/Music/music';
-import Settings from '../Components/Settings/settings';
-
-import route from '../route';
+import NavItem from '../Components/Navbar/NavItem/navItem';
 
 import './App.css';
 
 function App() {
```

Comment:
Consider using either class or arrow function notation for your components.

Reply:
I changed that